antiplag/tasks.py

```diff
@@ -39,8 +41,15 @@ def process_documents(submission_id):
 
     # update submission status
     submission.status = Submission.SubmissionStatus.PROCESSED
-    submission.save()
 
+    # send email when done
+    send_mail('Tvoja kontrola bola dokončená',
+        'Pozri si výsledok svojej kontroly na: antiplag.sk/result/' + str(submission.id) + "/" ,
+        'antiplag@antiplag.sk',
+        [submission.email],
+        fail_silently=False)
+
+    submission.save()
```

Comment:
What if sending the email fails? The submission will stay in the 'PROCESSING' forever. Shouldn't we reverse the order of sending email - saving submission status?